<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>keyconf</class>
 <widget class="QDialog" name="keyconf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>32</y>
     <width>185</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>CwKeyer Config</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>32</y>
     <width>177</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>( cwdaemon )</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonReturn">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>696</y>
     <width>365</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>96</y>
     <width>361</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Set your choosed  keydevice</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>64</y>
     <width>397</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineKeyPort">
   <property name="geometry">
    <rect>
     <x>148</x>
     <y>128</y>
     <width>237</width>
     <height>36</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>240</red>
         <green>240</green>
         <blue>240</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>240</red>
         <green>240</green>
         <blue>240</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>244</red>
         <green>244</green>
         <blue>244</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>176</y>
     <width>393</width>
     <height>24</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>11</red>
         <green>3</green>
         <blue>254</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>11</red>
         <green>3</green>
         <blue>254</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>146</red>
         <green>145</green>
         <blue>144</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>( See the provided README for more details ! )</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>212</y>
     <width>133</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>308</y>
     <width>249</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sidetone      ( 300 .. 1000 Hz )</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineSideton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>300</y>
     <width>85</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>244</y>
     <width>249</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sound  off              ( OFF = 0 )</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineSoundOnOff">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>236</y>
     <width>85</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>532</y>
     <width>253</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Wordmode              ( OFF/ON )</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineWordmode">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>524</y>
     <width>85</width>
     <height>36</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>235</red>
         <green>235</green>
         <blue>235</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>235</red>
         <green>235</green>
         <blue>235</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>244</red>
         <green>244</green>
         <blue>244</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>404</y>
     <width>249</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Weight                  ( -50 ... 50 )</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineWeight">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>396</y>
     <width>85</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>564</y>
     <width>249</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>UDP port</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineUdpPort">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>556</y>
     <width>85</width>
     <height>36</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>235</red>
         <green>235</green>
         <blue>235</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>235</red>
         <green>235</green>
         <blue>235</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>244</red>
         <green>244</green>
         <blue>244</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>436</y>
     <width>253</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PTT keying              ( OFF/ON )</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="linePttKeying">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>428</y>
     <width>85</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>596</y>
     <width>253</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tune x seconds   ( limit = 10 )</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineTuneX">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>588</y>
     <width>85</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>628</y>
     <width>249</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Band Index    ( See README )</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineBandIndex">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>620</y>
     <width>85</width>
     <height>36</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>235</red>
         <green>235</green>
         <blue>235</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>235</red>
         <green>235</green>
         <blue>235</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>244</red>
         <green>244</green>
         <blue>244</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>372</y>
     <width>249</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sound device ( See README )</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineSoundDev">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>364</y>
     <width>85</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>660</y>
     <width>249</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Reset to default     Values = 0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineReset">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>652</y>
     <width>85</width>
     <height>36</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>235</red>
         <green>235</green>
         <blue>235</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>235</red>
         <green>235</green>
         <blue>235</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>244</red>
         <green>244</green>
         <blue>244</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>500</y>
     <width>253</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>SSB signal from Mic. or Scard</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineSSBsig">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>492</y>
     <width>85</width>
     <height>36</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>235</red>
         <green>235</green>
         <blue>235</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>235</red>
         <green>235</green>
         <blue>235</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>244</red>
         <green>244</green>
         <blue>244</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>468</y>
     <width>253</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PTT on delay       ( 0 .. 50 ms )</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="linePttDelay">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>460</y>
     <width>85</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>272</y>
     <width>249</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Speed              ( 5 ... 60 wpm )</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineSpeed">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>268</y>
     <width>85</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>340</y>
     <width>257</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Soundcard volume  ( 0...100 )</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineSoundVolume">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>332</y>
     <width>85</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>136</y>
     <width>85</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Keyport</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
